change/@ni-nimble-components-71301602-90ef-49ef-8af6-558d62af1494.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we have a work item to track these documentation gaps?
1. storybook examples for different built-in column types
2. how to add a new column type 

I'm sure our clients will want more than what we have in the spec and storybook right now. Number 2 is obviously not needed as part of this PR. For 1 it could be useful to define our column docs pattern now but I'm also ok deferring it.

Reply:
Yeah, I'd rather defer even 1 for now, as this PR is a major blocker to other work.